static/docs/commands-reference/status.md

```diff
@@ -53,17 +54,30 @@ be rerun if `dvc repro` were executed.
 If instead, differences are detected, `dvc status` lists those changes. For each
 DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
 differ are listed. For each item listed, either the file name or the checksum is
-shown, and additionally a status word is shown describing the change:
+shown, and additionally a status word is shown describing the changes in
+checksum or status for both local and cloud workspace:
 
 - For the local workspace:
-  - _changed_ means the named file has changed
+
+  - _changed checksum_ means actual checksum of the DVC-file doesn't match the
+    one specified in it in DVC-files
+  - _always changed_ means that this is a special DVC-file with no dependencies,
+    which is considered always changed
+  - _new_ output is displayed when a new dependeny or output state is added to
```

Comment:
remove first output - it's confusing since we use output in two different meanings in the same sentence

Reply:
same below